samples/debug/ppi_trace/README.rst

```diff
@@ -0,0 +1,70 @@
+.. _ppi_trace_sample:
+
+PPI trace
+#########
+
+The PPI trace sample shows how to trace hardware events on GPIO pins.
+It uses the :ref:`ppi_trace` module.
+
+Overview
+********
+
+The sample initializes trace pins to observe the following hardware events:
+
+* RTC Compare event (``NRF_RTC_EVENT_COMPARE_1``)
+* RTC Tick event (``NRF_RTC_EVENT_TICK``)
+* Low frequency clock (LFCLK) Started event (``NRF_CLOCK_EVENT_LFCLKSTARTED``)
+* Radio activity during *Bluetooth* advertising (available only for Bluetooth capable devices)
+
+The sample sets up a :ref:`zephyr:counter_interface` to generate an event every 50 ms.
+Initially, RTC runs on RC low frequency (lower precision) as clock source.
+When the crystal is ready, it switches seamlessly to crystal (precise) as clock source.
+When the low-frequency crystal is ready, an ``NRF_CLOCK_EVENT_LFCLKSTARTED`` event is generated.
+
+
+Requirements
+************
+
+* One of the following development boards:
+
+  * nRF9160 DK board (PCA10090)
+  * nRF52840 Development Kit board (PCA10056)
+  * nRF52 Development Kit board (PCA10040)
+* Logic analyzer
+
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/debug/ppi_trace`
+
+.. include:: /includes/build_and_run.txt
+
+
+Testing
+=======
+
+After programming the sample to your board, test it by performing the following steps:
+
+1. Connect a logic analyzer to the pins that are used for tracing.
+   Check the sample configuration for information about which pins are used.
+   To do so in |SES|, select **Project** > **Configure nRF Connect SDK Project** and navigate to **PPI trace pins configuration**.
+#. Observe that the pin that is tracing the RTC Tick event is toggling.
+#. Measure the pin period of the pin that is tracing the RTC Compare event before and after the LFCLK Started event is generated.
```

Comment:
What is the "pin period" resp. does this make sense?

Reply:
maybe something like:
`Measure the typical time between two consecutive toggling of the pin that is tracing the RTC Compare event before and after the LFCLK Started event is generated. Observe that the precision increases when the low-frequency crystal is started.`

Reply:
OK, updated. :)